packages/ui/src/Center/Center.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Center as CenterComponent } from '.';
+
+type Center = typeof CenterComponent;
+
+const meta: Meta<Center> = {
+  argTypes: {
+    tag: {
+      control: 'text',
+    },
```

Comment:
```suggestion
    tag: {
      control: 'select',
      option: ["div", "span", "button"]
    },
```

다음과 같이 몇 가지 태그들을 넣어 select로 선택할 수 있게 하면 좋을 것 같습니다!

Reply:
태그들이 너무 많다보니 text를 사용하였는데 select를 통해 선택할 수 있게 하면 더 좋을것 같네요! 바꾸겠습니다.